src/lib/auth/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const COOKIE_AUTH_NAME = "Session";
```

Comment:
Let's keep cookies named using kebab-case, additional using purely `session` may be quite misleading as our auth is token-based. It is recommended that this be renamed to something more accurate like `user-token`.